### DISCUSSION TITLE
Add link to the app's repository in Contribution info links

### DISCUSSION THREAD

**Discussion by Cwpute** [FossifyOrg/General-Discussion#152] [Ideas]
Currently the app info page is great for getting infwrmation about the app, lists all known contributors, but isn't specifically asking for contributions

There is already a link to the organisation's Github page in the Social Network section, which is great for, well, networking, and being social.
But i'd like to see a link to the app's project page as well so people can contribute by sharing bugs and suggestions. It would be under the Help Us section, and i can see it either labelled "Project page" or "Issues and suggestions".

This would be more explicit regarding the collaborative-nature of this app, and would redirect users having problems with the app more efficiently i believe, not to mention future potential contributors.

**Comment by naveensingh** [comment]
I agree. It'll be added.

**Reply by naveensingh** [reply]
There is a "Known issues" option now (rolling out slowly in all apps) but there should also be a "Source code" option that directly links to each app's repository.
